test/parallel/test-fs-watch.js

```diff
@@ -64,7 +64,7 @@ for (const testCase of cases) {
       assert.strictEqual(eventType, 'change');
     assert.strictEqual(argFilename, testCase.fileName);
 
-    watcher.start();  // should not crash
+    watcher.start('should not crash');  // should not crash
```

Comment:
this is not the same, why this change?
It seems we are not exposing `start`  at all, see https://nodejs.org/api/fs.html#fs_class_fs_fswatcher.
I think we should add it to the docs.

Reply:
Looks like this should be `watcher.start(testCase[testCase.field])`?

Reply:
Looking at https://github.com/nodejs/node/issues/17430 and https://github.com/nodejs/node/pull/17432  I think this line should be either:

1. `watcher.start(testCase[testCase.field])`, we enforce that a filename must be passed to `.start()`
2. `watcher.start()`, and return silently in `FSWatcher.prototype.start()` if the filename is undefined (current behavior).

Reply:
Umm, looking more closely I think `.start()` is accepted the second time it is called (i.e. the wrap is already initialized to watch on a file), so in JS land we should check if the wrap is initialized before calling `validatePath()`. The easiest way to do it is to store a field in the watcher in `.start()` in JS land, echoing what's being done in the C++ land.

Reply:
Or, if we don't want to return silently when someone tries to call `.start()` again on the initialized watcher:

```
if (!isInitialized) {
  validatePath(filename)
} else {
  throw errors.Error('ERR_FS_EVENT_STARTED');  // A new Error indicating this has been started.
}
```